src/library/scala/annotation/restricted.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Scala (https://www.scala-lang.org)
+ *
+ * Copyright EPFL and Lightbend, Inc.
+ *
+ * Licensed under Apache License 2.0
+ * (http://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package scala.annotation
+
+import scala.annotation.meta._
+
+object RestrictedSeverity {
+  final val info    = "info"
+  final val warning = "warning"
+  final val error   = "error"
+  final val none    = "none"
+}
+
+object RestrictedLabel {
+  final val deprecated   = "deprecated"
+  final val internalOnly = "internalOnly"
+  final val apiMayChange = "apiMayChange"
```

Comment:
I think a "mistake" label for methods whose use indicates a coding mistake would be nice